src/commands/derive_parser.rs

```diff
@@ -0,0 +1,36 @@
+use crate::utils::auth_token::AuthToken;
+use crate::utils::value_parsers::kv_parser;
+use clap::{command, value_parser, ArgAction::SetTrue, Parser, Subcommand};
+
+use super::send_metric::SendMetricArgs;
+
+#[derive(Parser)]
+pub(super) struct SentryCLI {
```

Comment:
can you write in the PR description (and thus in the final commit message) that this is a new way of writing future CLI commands (using the derives) since we were using a different pattern before?